hr_holidays_half_day/views/hr_holidays_view.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <!-- Leave Request -->
+        <record id="edit_holiday_new_inherited" model="ir.ui.view">
+            <field name="model">hr.holidays</field>
+            <field name="inherit_id" ref="hr_holidays.edit_holiday_new" />
+            <field name="arch" type="xml">
+                <form position="attributes">
+                    <attribute name="create">false</attribute>
+                </form>
+                <field name="date_from" position="attributes">
+                    <attribute name="context">{'employee_id': employee_id, 'holiday_status_id': holiday_status_id}</attribute>
+                    <attribute name="readonly">1</attribute>
+                </field>
+                <field name="date_to" position="attributes">
+                    <attribute name="context">{'employee_id': employee_id, 'holiday_status_id': holiday_status_id}</attribute>
```

Comment:
@houssine78 the employee_id is given with the context

Reply:
not sure though that this is passed to the onchange method.

Reply:
I verified and it is passed to the onchange methods